aws-lambda-java-runtime-interface-client/test/integration/codebuild/buildspec.os.alpine.yml

```diff
@@ -41,6 +41,14 @@ phases:
       - >
         echo "COPY ${SCRATCH_DIR}/aws-lambda-rie /usr/bin/aws-lambda-rie" >> \
           "${SCRATCH_DIR}/Dockerfile.function.${OS_DISTRIBUTION}.tmp"
+      - >
+        if [[ -z "${DOCKERHUB_USERNAME}" && -z "${DOCKERHUB_PASSWORD}" ]];
+        then
+            echo "DockerHub credentials not set as CodeBuild environment variables. Continuing without docker login."
+        else
+            echo "Performing DockerHub login . . ."
+            docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD
+        fi
```

Comment:
I recommend doing this before line 28 (maybe even in the `install` phase) as the RIC build also pulls in DockerHub images to build the JNIs

Reply:
Correct. Modified in next commit.